fitbenchmarking/results_output.py

```diff
@@ -14,7 +14,7 @@
 
 from fitbenchmarking.utils.logging_setup import logger
 from fitbenchmarking.resproc import visual_pages
-from fitbenchmarking.utils import create_dirs, options, misc
+from fitbenchmarking.utils import create_dirs
```

Comment:
Should these be in alphabetical order as well?

Reply:
yes